sasl/hmac.go

```diff
@@ -0,0 +1,230 @@
+package sasl
+
+import (
+	"bytes"
+	"crypto/cipher"
+	"crypto/des"
+	"crypto/hmac"
+	"crypto/md5"
+	"crypto/rc4"
+	"encoding/binary"
+	"fmt"
+)
+
+var (
+	errEmptyBuffer                = fmt.Errorf("ERROR: Empty buffer")
```

Comment:
Remove `ERROR: ` from all of these strings and make the initial letters lowercase.